src/AzSK.ADO/AzSKADOInfo/AzSKADOInfo.ps1

```diff
@@ -151,6 +157,16 @@ function Get-AzSKADOInfo
 					{
 						Write-Host -ForegroundColor Yellow "AttestationInfo support is yet to be implemented."
 					}
+					UserInfo
+					{
+						if($ProjectNames -eq "*" -or $ProjectNames -match "\,") {
+							Write-Host "Parameter set 'UserInfo' only accepts scan for single project, try again with one project name (or no project name for organization level scan)." -ForegroundColor Red
```

Comment:
(or without -projectname parameter for organization level info)

Reply:
As PrincipalName param is not mandatory, a check should be added in UserInfo block to validate it.